client/driver/docker.go

```diff
@@ -1924,6 +1925,21 @@ func (h *DockerHandle) run() {
 		h.logger.Printf("[ERR] driver.docker: failed to inspect container %s: %v", h.containerID, ierr)
 	} else if container.State.OOMKilled {
 		werr = fmt.Errorf("OOM Killed")
+		labels := []metrics.Label{
```

Comment:
for me the interesting labels would be job, group and task :)

Reply:
Yes for me too, but i don't know how to access them from here.

Reply:
Oh really nice @jvrplmlmn Ill update this PR as soon as I get back from my vacations.